l10n_it_fatturapa/bindings/__init__.py

```diff
@@ -17,5 +17,7 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-
-from . import fatturapa_v_1_1
+try:
+    from . import fatturapa_v_1_1
+except ImportError:
+    pass # Module is not installed, avoid init error
```

Comment:
Yep that's it, there are a lot of statements (not only imports) that need
to be guarded against. You are right, a comment would be nice, I'll add
one.

On Mon, Nov 30, 2015, 18:20 Lorenzo Battistini <notifications@github.com>
wrote:

> In l10n_it_fatturapa/bindings/__init__.py
> <https://github.com/OCA/l10n-italy/pull/164#discussion_r46173170>:
>
> > @@ -17,5 +17,7 @@
> >  #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
> >  #
> >  ##############################################################################
> > -
> > -from . import fatturapa_v_1_1
> > +try:
> > +    from . import fatturapa_v_1_1
> > +except ImportError:
> > +    pass # Module is not installed, avoid init error
>
> Why this? When you use this module, file fatturapa_v_1_1.py is always
> present
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OCA/l10n-italy/pull/164/files#r46173170>.
>